core/src/Cuda/Kokkos_CudaSpace.cpp

```diff
@@ -51,6 +51,15 @@
 #include <algorithm>
 #include <atomic>
 
+// The purpose of the following variable is to allow a state-based choice
+// for pinning UVM allocations to the CPU. For now this is considered
+// an experimental debugging capability - with the potential to work around
+// some CUDA issues.
+#ifdef KOKKOS_IMPL_DEBUG_CUDA_PIN_UM_TO_HOST
+bool kokkos_impl_cuda_pin_um_to_host_v = false;
```

Comment:
A bit of a nit-pick comment, but the `um` naming here vs `UVM` in the unit test file could be made consistent.

Reply:
yeah interesting question this. NVIDIA insists on now calling it UM (Unified Memory) since it is not "Virtual" anymore. But we could rename those to align with current naming in Kokkos.